lib/screens/app/wallet/scan.dart

```diff
@@ -183,7 +200,11 @@ class _ScanState extends State<Scan> {
           mainAxisAlignment: MainAxisAlignment.center,
           children: [
             OutlineButton(
-              child: Text('Try Again'),
+              child: Text(
+                'Try Again',
```

Comment:
translations?

Reply:
Volver a probar

Reply:
hummmmm. How about 

`Probar` can be to try or to taste.
`Intentar` = `Try`

`Intentar Nuevamente`